hugo/content/memory/_index.md

```diff
@@ -35,69 +33,74 @@ The first 2 bits in a 32-bit address.
-| `QEMU_DEBUG`      | w     | w         |        | u8       |           | Debug console (only in QEMU)                                            |
-| `NAME0`           | r     | r         | 4B     | char[4]  | "tk1 "    | ID of core/stick, first part.                                           |
-| `NAME1`           | r     | r         | 4B     | char[4]  | "mkdf"    | ID of core/stick, second part.                                          |
-| `VERSION`         | r     | r         | 4B     | u32      | 1         | Version of core/stick.                                                  |
-| `SWITCH_APP`      | r/w   | r         | 1B     | u8       |           | Write anything here to trigger the switch to application mode. Reading returns `0` if TKey is in firmware mode, `0xffffffff` if in app mode. |
-| `LED`             | r/w   | r/w       | 1B     | u8       |           | Controls the RGB color of the status indicator LED on TKey. Bit 0 is Blue, bit 1 is Green, and bit 2 is Red LED. |
-| `GPIO`            | r/w   | r/w       | 1B     | u8       |           | Bits 0 and 1 contain the input level of GPIO 1 and 2. Bits 3 and 4 store the output level of GPIO 3 and 4. |
-| `APP_ADDR`        | r/w   | r         | 4B     | u32      |           | App load address, stored by firmware so app can find itself in memory.  |
-| `APP_SIZE`        | r/w   | r         | 4B     | u32      |           | App size, stored by firmware so app can read its own size.              |
-| `BLAKE2S`         | r/w   | r         | 4B     | u32      |           | Function pointer to a BLAKE2S function in the firmware.                 |
-| `CDI_FIRST`       | r/w   | r         | 32B    | u8[32]   |           | The computed Compound Device Identifier (CDI).                          |
-| `CDI_LAST`        |       | r         |        |          |           | Last word of CDI.                                                       |
-| `RAM_ASLR`        | w     | invisible | 4B     | u32      |           | Seed value for the RAM randomization.                                   |
-| `RAM_SCRAMBLE`    | w     | invisible | 4B     | u32      |           | Data scrambling seed value for the RAM.                                 |
-| `CPU_MON_CTRL`    | w     | w         | 4B     | u32      |           | Bit 0 enables CPU execution monitor. Can't be unset. Lock addresses.    |
-| `CPU_MON_FIRST`   | w     | w         | 4B     | u32      |           | First address of the area monitored for execution attempts.             |
-| `CPU_MON_LAST`    | w     | w         | 4B     | u32      |           | Last address of the area monitored for execution attempts.              |
-
-[^3]: Each word of the UDS can only be read *once* after TKey has started.
+All the constants in `tk1_mem.h` are prefixed with `TK1_MMIO_` but to
+save space we present them here without the prefix.
+
```

Comment:
Or drop MMIO.

Reply:
Is this a suggestion to drop the `MMIO` prefix in `tk1_mem.h` in tillitis-key1 and tkey-libs?

Reply:
Yes. Dropping the MMIO_ prefix here, but not in header files etc means that we create a confusion, even if we have that note.

As stated before, I don't fancy the MMIO term (even if I know it probably is a lost battle). But to me it does not make sense calling things that clearly are not I/O MMIO. The API names in the cores does not have MMIO_-prefix. It has been added later. Which I think cause confusion too. (If we are to be pedantic about ASLR, we should be it with MMIO too.)

Either have MMIO_ everywhere or nowhere, even in this document though it is considered cumbersome. And This would also mean that we should update the whole API space for all cores with MMIO_.